pkg/kubectl/cmd/version.go

```diff
@@ -46,36 +59,92 @@ func NewCmdVersion(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	}
 	cmd.Flags().BoolP("client", "c", false, "Client version only (no server required).")
 	cmd.Flags().BoolP("short", "", false, "Print just the version number.")
+	// default behavior is std.
+	cmd.Flags().String("output", "std", "output format, options available are yaml and json")
```

Comment:
i was going back and forth on this. I settled on std simply because i thought empty string would be a bit more difficult to understand? Not sure, but i'll change it to empty string if you are ok with that?